grapher/mapCharts/AnnotationGenerator.ts

```diff
@@ -630,6 +545,10 @@ function getLabelInfo(
                     if (u in allPoints) allPoints[u] = allPoints[u] + 1
                     else allPoints[u] = 1
```

Comment:
I can see this code is repeated, can it maybe be extracted into a helper function?

```ts
                for (const temp of regionPath) {
                    const o = [
                        Number(temp.substring(0, temp.indexOf(","))),
                        Number(temp.substring(temp.indexOf(",") + 1)),
                    ]
                    tempPoints.push(o)
                    const u = o[0].toString() + "," + o[1].toString()
                    if (u in allPoints) allPoints[u] = allPoints[u] + 1
                    else allPoints[u] = 1
                }
```